tests/php/Controllers/Root/RootController/DevActionTest.php

```diff
@@ -0,0 +1,446 @@
+<?php
+
+use App\Http\Controllers\Root\RootController;
+use Illuminate\Http\Request;
+
+/**
+ * RootControllerクラスのdevActionのテストを行う.
+ *
+ * @SuppressWarnings(PHPMD.TooManyPublicMethods)
+ */
+class DevActionTest extends TestCase
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)